cli/flox/src/commands/push.rs

```diff
@@ -34,7 +34,7 @@ pub struct Push {
     dir: Option<PathBuf>,
 
     /// FloxHub owner to push environment to (default: current FloxHub user)
-    #[bpaf(long, short, argument("owner"))]
+    #[bpaf(long("owner"), long("org"), short, argument("owner"))]
```

Comment:
Note: additional long arguments don't get rendered in `--help`.

@jennymahmoudi do you foresee that being a problem?

Reply:
Thanks for calling that out. It's fine to include the `--org` alias in the description only.